pom.xml

```diff
@@ -28,12 +28,14 @@
     </parent>
 
     <groupId>org.glassfish</groupId>
```

Comment:
Hmm, maybe group should be `org.glassfish.concurro`?

Reply:
Yes indeed, that was the intention. Thanks for catching it!

Reply:
It is done!